### PR TITLE
ASP.NET Core

### PR DESCRIPTION
**Pull request by forinov22**
console app reworked into asp.net core app, added brands controller, model binding, model validation

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,9 +3,6 @@
 ##
 ## Get latest from https://github.com/github/gitignore/blob/main/VisualStudio.gitignore
 
-bin
-obj
-
```
Comment (Katletos) [inline .gitignore:8]: better to add via dotnet buildin tool

File: DbFirstApp/Controllers/BrandsController.cs [hunk 2]
```diff
@@ -0,0 +1,119 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using DbFirstApp.Data;
+using DbFirstApp.Models;
+
+namespace OnlineShop.Controllers
+{
+    [Route("api/[controller]")]
```
Comment (Katletos) [inline DbFirstApp/Controllers/BrandsController.cs]: define route explicit

File: DbFirstApp/Controllers/BrandsController.cs [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+    public class BrandsController : ControllerBase
+    {
+        private readonly OnlineShopContext _context;
+
+        public BrandsController(OnlineShopContext context)
+        {
+            _context = context;
+        }
+
+        // GET: api/Brands
```
Comment (Katletos) [inline DbFirstApp/Controllers/BrandsController.cs]: delete comments, I hate this

File: DbFirstApp/Controllers/BrandsController.cs [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+        {
+          if (_context.Brands == null)
+          {
+              return NotFound();
+          }
+            return await _context.Brands.ToListAsync();
+        }
+
+        // GET: api/Brands/5
+        [HttpGet("{id}")]
```
Comment (Katletos) [inline DbFirstApp/Controllers/BrandsController.cs]: Does it work? use `FromRoute` annotation

File: DbFirstApp/Controllers/BrandsController.cs [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+        public BrandsController(OnlineShopContext context)
+        {
+            _context = context;
+        }
+
+        // GET: api/Brands
+        [HttpGet]
+        public async Task<ActionResult<IEnumerable<Brand>>> GetBrands()
+        {
+          if (_context.Brands == null)
```
Comment (Katletos) [inline DbFirstApp/Controllers/BrandsController.cs]: explain why you check for null

File: DbFirstApp/Startup.cs [hunk 6]
```diff
@@ -0,0 +1,22 @@
+﻿using DbFirstApp.Data;
+
+namespace OnlineShop
+{
+    public class Startup
```
Comment (Katletos) [inline DbFirstApp/Startup.cs]: fuck, in .net 7 you don't need a startup class. Read about and fix

**Review by Katletos — CHANGES_REQUESTED** [review]

File: DbFirstApp/Controllers/BrandsController.cs (hunk 4 above)
Comment (forinov22) [reply DbFirstApp/Controllers/BrandsController.cs]: yes, it is

**Pull request closed without merging**